konduit-serving-pipeline/src/main/java/ai/konduit/serving/pipeline/impl/pipeline/graph/AnyStep.java

```diff
@@ -31,14 +33,14 @@
  *
  * @author Alex Black
  */
-@EqualsAndHashCode(callSuper = true)
+@Data
```

Comment:
As already discussed in the previous PR review: We need to keep `@EqualsAndHashCode(callSuper = true)`. This same reason applies to every single one of these annotations that you've removed (not just the ones I've flagged)